dali/pipeline/operators/reader/loader/indexed_file_loader.h

```diff
@@ -104,6 +104,7 @@ class IndexedFileLoader : public Loader<CPUBackend, Tensor<CPUBackend>> {
     std::vector<std::string> index_uris = options.GetRepeatedArgument<std::string>("index_path");
     ReadIndexFile(index_uris);
     size_t num_indices = indices_.size();
+    DALI_ENFORCE(num_indices != 0, "Content of index files should not be empty");
```

Comment:
`DALI_ENFORCE(!indices_.empty())` then the error message and the line before are not needed

Reply:
Not really, error telling you `!indices_.empty()` is not telling the user anything, while `Content of index files should not be empty` does.

Reply:
Alright, but `!indices.empty()` is still more clear for the reader than getting the size and then check if it is different than 0

Reply:
Done